dataactcore/utils/jobQueue.py

```diff
@@ -28,11 +29,13 @@ def __init__(self, job_queue_url="localhost"):
         @self.jobQueue.task(name='jobQueue.enqueue')
         def enqueue(jobID):
             # Don't need to worry about the response currently
+            CloudLogger.log("Adding job {} to the queue".format(str(jobID)))
```

Comment:
We should also output the response json on line 39 below. That would provide us with a more in-depth trace if any errors occur. The `queueURL` wouldn't be too helpful as it was just refer to itself (localhost). Validator URL may be helpful but that can just be derived from the config.